Packs/CommonScripts/Scripts/script-DeleteConext.yml

```diff
@@ -16,6 +16,7 @@ script: >
 
   var shouldDeleteAll = (args.all === 'yes');
 
+
```

Comment:
Don't forget to remove this
```suggestion
```

Reply:
yes, sure. I am waiting to the build to be successfully done and the I'll remove that line.